project/src/main/java/edu/depaul/cdm/se452/project/model/CourseRepository.java

```diff
@@ -5,7 +5,7 @@
 import org.springframework.data.repository.CrudRepository;
 
 //repository class to help perform CRUD operations on the courses table
-public interface CourseRepository extends CrudRepository<Course, Long> {
+public interface CourseRepository extends CrudRepository<Course, Integer> {
```

Comment:
@crotert would you double check that this doesn't mess with your functionality before merging? I believe the second type here is supposed to be the ID type of the model. In the case of `Course`, that would be `int` or `Integer`

Reply:
Thanks for catching that Michael.  Looks good.